Warehouse.Domain/Enums/OrderStatusName.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Warehouse.Domain.Enums;
+
+public enum OrderStatusName
+{
+    Pending,
```

Comment:
Use "reserved null enum pattern". For all your enums!
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008